### PR TITLE
[DOC-8610] Document ISO 27001 and 27017 compliance

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-8610] Document ISO 27001 and 27017 compliance

### CODE REVIEW

**Comment by adbrenn** [comment]
@mdlinville - there is one typo in the last sentence:

CockroachDB Dedicated meets the requirements of ISO 27001 and ISO **1**7017. Other than that LGTM

File: src/current/cockroachcloud/compliance.md [hunk 1]
```diff
@@ -16,3 +16,5 @@ When configured correctly, CockroachDB {{ site.data.products.dedicated }} meets
...
 
 - **Health Insurance Portability and Accountability Act (HIPAA)**: The Health Insurance Portability and Accountability Act of 1996, commonly referred to as _HIPAA_, defines standards for the storage and handling of personally-identifiable information (PII) related to patient healthcare and health insurance. When configured appropriately for [PCI DSS Compliance]({% link cockroachcloud/pci-dss.md %}), CockroachDB {{ site.data.products.dedicated }} advanced also meets the requirements of HIPAA.
+
+- **ISO 27001** and **ISO 27017**: ISO 27001 and ISO 27017 define an international standards for managing information security. ISO 27001 is a general standard, and ISO 27017 is a standard specific to cloud service providers and environments, to improve security and reduce the risk of security problems. These standards are governed jointly by the [International Organization for Standardization (ISO)](https://www.iso.org/home.html) and the [International Electrotechnical Commission (IEC)](https://www.iec.ch/homepage). CockroachDB {{ site.data.products.dedicated }} meets the requirements of ISO 27001 and ISO 17017.
```
Comment (mdlinville) [inline src/current/cockroachcloud/compliance.md:20]:
```suggestion
- **ISO 27001** and **ISO 27017**: ISO 27001 and ISO 27017 define an international standards for managing information security. ISO 27001 is a general standard, and ISO 27017 is a standard specific to cloud service providers and environments, to improve security and reduce the risk of security problems. These standards are governed jointly by the [International Organization for Standardization (ISO)](https://www.iso.org/home.html) and the [International Electrotechnical Commission (IEC)](https://www.iec.ch/homepage). CockroachDB {{ site.data.products.dedicated }} meets the requirements of ISO 27001 and ISO 27017.
```

**Comment by mdlinville** [comment]
> @mdlinville - there is one typo in the last sentence:
> 
> CockroachDB Dedicated meets the requirements of ISO 27001 and ISO **1**7017. Other than that LGTM

Done!

**Review by adbrenn — APPROVED** [review]
LGTM

File: src/current/cockroachcloud/compliance.md [hunk 2]
```diff
@@ -16,3 +16,5 @@ When configured correctly, CockroachDB {{ site.data.products.dedicated }} meets
...
 
 - **Health Insurance Portability and Accountability Act (HIPAA)**: The Health Insurance Portability and Accountability Act of 1996, commonly referred to as _HIPAA_, defines standards for the storage and handling of personally-identifiable information (PII) related to patient healthcare and health insurance. When configured appropriately for [PCI DSS Compliance]({% link cockroachcloud/pci-dss.md %}), CockroachDB {{ site.data.products.dedicated }} advanced also meets the requirements of HIPAA.
+
+- **ISO 27001** and **ISO 27017**: ISO 27001 and ISO 27017 define an international standards for managing information security. ISO 27001 is a general standard, and ISO 27017 is a standard specific to cloud service providers and environments, to improve security and reduce the risk of security problems. These standards are governed jointly by the [International Organization for Standardization (ISO)](https://www.iso.org/home.html) and the [International Electrotechnical Commission (IEC)](https://www.iec.ch/homepage). CockroachDB {{ site.data.products.dedicated }} meets the requirements of ISO 27001 and ISO 27017.
```
Comment (shannonbradshaw) [inline src/current/cockroachcloud/compliance.md:20]:
```suggestion
- **ISO 27001** and **ISO 27017**: ISO 27001 and ISO 27017 define international standards for managing information security. ISO 27001 is a general standard, and ISO 27017 is a standard specific to cloud service providers and environments, to improve security and reduce the risk of security problems. These standards are governed jointly by the [International Organization for Standardization (ISO)](https://www.iso.org/home.html) and the [International Electrotechnical Commission (IEC)](https://www.iec.ch/homepage). CockroachDB {{ site.data.products.dedicated }} meets the requirements of ISO 27001 and ISO 27017.
```

Comment (shannonbradshaw) [reply src/current/cockroachcloud/compliance.md:20]: > , to improve security and reduce the risk of security problems

This clause seems unnecessary.

**Review by shannonbradshaw — APPROVED** [review]
LGTM. One small fix + a suggested cut.

File: src/current/cockroachcloud/compliance.md (hunk 2 above)
Comment (mdlinville) [reply src/current/cockroachcloud/compliance.md:20]: Edited your edit. Thanks!

**Pull request merged**